packages/frontend/src/components/CreateUserForm.tsx

```diff
@@ -17,6 +18,10 @@ const CreateUserForm: FC<Props> = ({ isLoading, onSubmit }) => {
     const methods = useForm<CreateUserArgs>({
         mode: 'onSubmit',
     });
+    const { health } = useApp();
+
+    const disablePasswordAuth = health.allowPasswordAuthentication;
```

Comment:
@nathaliabj 

```suggestion
    const disablePasswordAuth = health.data.allowPasswordAuthentication;
```

Also, we should hide the entire section that creates an user with user and password. It should only have the google auth button.

Reply:
Also, `disablePasswordAuth` doesn't seem the correct name. Shouldn't we use `allowPasswordAuthentication` and display the fields if the value is true ?